raiden/accounts.py

```diff
@@ -59,13 +59,15 @@ def __init__(self, keystore_path=None):
                         with open(fullpath) as data_file:
                             data = json.load(data_file)
                             self.accounts[str(data['address']).lower()] = str(fullpath)
-                    except (ValueError, KeyError, IOError) as ex:
+                    except (KeyError, IOError, json.decoder.JSONDecodeError) as ex:
```

Comment:
`open()` in python 3.6 can throw an `OSError` according to the [docs](https://docs.python.org/3.6/library/functions.html#open).

The `OSError` contains the error text in the `strerror` attribute.

Reply:
Also `open()` can throw a `PermissionError` if the user does not have adequate permissions to open the file

Reply:
It's sufficient to just catch `OSError` since `PermissionError` is a subclass and `IOError` is (since 3.3) a backward compatibility alias of `OSError`

Reply:
Thank you for the note @ulope! So yeah @gcarq please do that.